### PR TITLE
[AAP-22947] e2e jobTemplateSurveys tests

### PR DESCRIPTION
**Pull request by jerabekjiri**
Issue: [AAP-22947](https://issues.redhat.com/browse/AAP-22947)

### CODE REVIEW

**Review by akus062381 — CHANGES_REQUESTED** [review]
When I ran the tests locally, they mostly looked great! However I did see an error with this test:
image

and the error was:
image

File: cypress/e2e/awx/resources/jobTemplateSurveys.cy.ts [hunk 1]
```diff
@@ -12,119 +12,414 @@ describe('Job Templates Surveys', function () {
...
+      });
+
+      cy.clickButton('Finish');
+
+      cy.intercept('POST', awxAPI`/job_templates/${jobTemplate.id.toString()}/launch/`).as(
+        'postLaunch'
+      );
+      cy.clickButton(/^Finish/);
+      cy.wait('@postLaunch')
+        .its('response.body.id')
```
Comment (akus062381) [inline cypress/e2e/awx/resources/jobTemplateSurveys.cy.ts:405]: At this point in the test, I am seeing this error:
image

**Review by akus062381 — CHANGES_REQUESTED** [review]
Looks like this test just needs a bit of tweaking :-)

File: cypress/e2e/awx/resources/jobTemplateSurveys.cy.ts (hunk 1 above)
Comment (jerabekjiri) [reply cypress/e2e/awx/resources/jobTemplateSurveys.cy.ts:405]: yes, this was caused by https://github.com/ansible/ansible-ui/pull/2241, rerunning and should be fixed now

**Pull request merged**